### DISCUSSION TITLE
How to do search with mongodb?

### DISCUSSION THREAD

**Discussion by mr0b0t0** [prisma/prisma#14983] [Q&A] [answered]
How to do a search with MongoDB and prisma?

Something like this with native mongo: `MyModel.find({$text: {$search: searchString}})`
Or, `LIKE` with SQL

I believe it's possible with `findRaw`

```
... .findRaw({
    filter: {
      $text: {
        $search: "...,
      },
    },
  })
```

**Comment by austincrim** [comment, marked as answer]
Hey @mr0b0t0 👋,

The `findRaw` query mentioned above is the way to do it. Prisma's `fullTextSearch` preview feature only applies to Postgres and MySQL for now.

Feel free to open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) asking for better text search support for MongoDB.

**Reply by janpio** [reply]
Already exists: https://github.com/prisma/prisma/issues/9413 - so 👍 on that would be welcome.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
